lib/plugins/interactiveCli/index.js

```diff
@@ -1,27 +1,17 @@
 'use strict';
 
-const _ = require('lodash');
 const inquirer = require('./inquirer');
 const initializeService = require('./initializeService');
 const setupAws = require('./setupAws');
 const tabCompletion = require('./tabCompletion');
 
 module.exports = class InteractiveCli {
   constructor(serverless) {
-    if (!process.stdin.isTTY) return;
-
-    const { processedInput } = serverless;
-    if (processedInput.commands.length) return;
-    if (!_.isEmpty(processedInput.options)) return;
-
-    // Expose customized inquirer, and setupAws configuration for other plugins
-    // setupAws is further customized by dashboard plugin
-    serverless.interactiveCli = { inquirer, awsSetupConfiguration: setupAws };
+    this.serverless = serverless;
 
-    // Enforce interactive CLI
-    processedInput.commands.push('interactiveCli');
     this.commands = {
```

Comment:
Now as we setup the command unconditionally. It lists in help as `interactiveCLI..... undefined`:

![Screenshot 2019-10-22 at 12 52 56](https://user-images.githubusercontent.com/122434/67279564-8cd7c480-f4cb-11e9-808c-f7de90dae72a.png)

Not sure exactly what's the best way to address it, but maybe adding `isHidden` property on command configuration and ensuring to skip all such commands in help listing is the way to go (?)

@pmuens what do you think?

Reply:
That seems reasonable. Implemented.